City-of-Pori/pori-events
# PORISD-51 Calendar additions

# Calendar additions
* Fixing the +/- alignment within the filter listing (responsive mode fixed as well)
* Adding a +- sign to filters with subareas and hobby categories (only non-functional approach / only visually)
* Adding a sub-area in search results.

# Technical
* Styling changes for events (areas -> pori = +/- sign added) and hobby categories (each parent item +/- added)
* .js App changes (in order to show proper sub-areas for events and hobbies

# Test
https://pori-events.dev.wunder.io/

* In the filtering see that +/- alignment is fixed (also in responsive mode)
* Go to events -> area -> click pori to see that pori have +/-  sign
* Go to hobbies -> categories -> click on any parent item to see +/-  sign for only parents
* See in search results list how items of sub-area are displayed. 

# Story
https://wunder.atlassian.net/browse/PORISD-51 - Additions to event/hobby-calendar (extended from PORI-489)
